operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/spec/IngressSpec.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.operator.crds.v2alpha1.deployment.spec;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.sundr.builder.annotations.Buildable;
+
+@Buildable(editableEnabled = false, builderPackage = "io.fabric8.kubernetes.api.builder")
+public class IngressSpec {
+
+    @JsonProperty("enabled")
+    private boolean ingress;
```

Comment:
Just a non-blocking nitpick (just in case you'd be making some other changes, no need to fix it otherwise).

I'd suggest to rename this field to better capture its meaning:
```suggestion
    private boolean enabled;
```

Reply:
@vmuzikar  overlooked initially, renamed. Thanks